src/host.js

```diff
@@ -94,7 +94,7 @@ Host._createIFrame = function createIFrame(src, frameId, height, allowFullScreen
 };
 
 Host._tryGetOrigin = function tryGetOrigin(url) {
-	if (url && url.indexOf('//') === 0) {
+	if (url && url.indexOf('//') === -1) {
```

Comment:
are there unit tests?

Reply:
yeah, and failing with this change. I like the comment you put regarding allow both cases, if the proceeding // is a legitimate case